Dockerfile

```diff
@@ -1,50 +1,37 @@
-FROM cm2network/steamcmd:root as build_stage
+FROM cm2network/steamcmd:root
 
 LABEL maintainer="davemc08@gmail.com"
 
-ENV STEAMAPPID 1348640 
-ENV STEAMAPP frozen-flame
-ENV STEAMAPPDIR "${HOMEDIR}/${STEAMAPP}-dedicated"
-
-COPY --chown="$USER:$USER" entry.sh "${HOMEDIR}/entry.sh"
-COPY --chown="$USER:$USER" entry.sh "${HOMEDIR}/tinientry.sh"
-
 RUN set -x \
-	# Install, update & upgrade packages
-	&& apt-get update \
-	&& apt-get install -y --no-install-recommends --no-install-suggests \
-		wget=1.21-1+deb11u1 \
-		ca-certificates=20210119 \
-		lib32z1=1:1.2.11.dfsg-2+deb11u2 \
-		tini=0.19.0-1 \
-		libc6-dev=2.31-13+deb11u5 \
-		file=1:5.39-3 \
-	&& mkdir -p "${STEAMAPPDIR}/FrozenFlame/Saved" \
-	# Add entry scripts
-	&& chmod +x "${HOMEDIR}/entry.sh" "${HOMEDIR}/tinientry.sh" \
-	&& chown -R "${USER}:${USER}" "${STEAMAPPDIR}"  \
-	# Clean up
-	&& rm -rf /var/lib/apt/lists/*
-
-
-FROM build_stage AS bullseye-base
+    && apt-get update \
+    && DEBIAN_FRONTEND=noninteractive apt-get install -y gosu --no-install-recommends\
+    && rm -rf /var/lib/apt/lists/*  \
+    && gosu nobody true
 
-ENV SERVER_NAME="FrozenFlameDedicated" \
-	RCON_PORT="" \
-	RCON_PASSWORD="password" \
-	STEAMCMD_UPDATE_ARGS=""
+RUN mkdir -p /config \
+ && chown steam:steam /config
 
-# Switch to user
-USER ${USER}
+COPY init.sh /
 
-WORKDIR ${HOMEDIR}
+COPY --chown=steam:steam *.ini run.sh /home/steam/
 
-EXPOSE 25575 \
```

Comment:
I think we should still be exposing the ports from the docker image itself vs the compose file. This should be easier to configure from an end user perspective.

Reply:
I am not agree with this idea.
I have some game servers who use the same port by default. when I change the configs, the port is still expose.
All my docker game server is with `network_mode: "host"` I don't know if I am the only one who do that.